From 646cd86cb8bb5078d3c421b3e8d1dc8bdefb115e Mon Sep 17 00:00:00 2001
From: Randy Li <randy.li@rock-chips.com>
Date: Tue, 25 Apr 2017 11:08:03 +0800
Subject: [PATCH] ARM: dts: rockchip: remove dev_mode from rk3288

Since there is no combo device at RK3288, no need to
use this property anymore.

Change-Id: I56434161c4167fc048e4956e97b29617367e28f6
Signed-off-by: Randy Li <randy.li@rock-chips.com>
---
 arch/arm/boot/dts/rk3288.dtsi | 2 --
 1 file changed, 2 deletions(-)

diff --git a/arch/arm/boot/dts/rk3288.dtsi b/arch/arm/boot/dts/rk3288.dtsi
index e0675ef3149d..5c850480b228 100644
--- a/arch/arm/boot/dts/rk3288.dtsi
+++ b/arch/arm/boot/dts/rk3288.dtsi
@@ -1398,7 +1398,6 @@
 		reset-names = "video_a", "video_h";
 		iommus = <&vpu_mmu>;
 		iommu_enabled = <1>;
-		dev_mode = <0>;
 		status = "disabled";
 		/* 0 means ion, 1 means drm */
 		allocator = <1>;
@@ -1439,7 +1438,6 @@
 		reset-names = "video";
 		power-domains = <&power RK3288_PD_HEVC>;
 		rockchip,grf = <&grf>;
-		dev_mode = <1>;
 		iommus = <&hevc_mmu>;
 		iommu_enabled = <1>;
 		status = "disabled";
-- 
2.35.3

